Sage/carbon
# Add caption prop to the Table component

The `Table` component now supports an optional `caption` prop which, if given, renders as a `<caption>` element within the table.

## Related Issues / Pull Requests
#1143 

## Screenshot

VoiceOver rotor showing tables with captions:

<img width="794" alt="rotor_tables" src="https://user-images.githubusercontent.com/52329/27233749-89f76184-52b2-11e7-93c6-90766a62a814.png">

## Testing Instructions
The `caption` is visually hidden but can be tested using:

* Browser dev tools e.g. element inspector
* VoiceOver on OS X - turn on with Cmd+F5, bring up the rotor using Ctrl+Option+U, use left / right arrows to find the Tables, and you should see the table listed via its caption.
